### ISSUE TITLE
"byteBuffer.nextLong is not a function" error when reading bigWig file

### ISSUE THREAD

**Issue by skushch** [igvteam/igv.js#384]
Hi,

I ran into an error trying to load a few bigWig tracks, here's one example:
https://s3-us-west-2.amazonaws.com/temporary-igv-bug-data/gse50747.rattus-norvegicus-81.bw

Initial load triggers the error:
```
TypeError: byteBuffer.nextLong is not a function
    at readTreeNode (igv-1.0.8.js:3312)
    at new igv.BPTree (igv-1.0.8.js:3280)
    at igv-1.0.8.js:3741
    at <anonymous>
```

If you dismiss the error dialog and try to move the canvas you'll get another error:
```
TypeError: Cannot read property 'dictionary' of undefined
    at igv-1.0.8.js:3860
    at <anonymous>
```

The Java IGV client is able to open the same file without any issues.

Thanks,
Sergey

**Comment by jrobinso** [comment]
I suspect this is fixed in master (beta),  if you are able to could you try it there?   Note that the beta files required "underscore".    

https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js
http://igv.org/web/beta/igv-beta.js
http://igv.org/web/beta/igv.css

**Comment by jrobinso** [comment]
BTW I can't test with that URL because CORS headers are not enabled.

**Comment by skushch** [comment]
You're right, the beta version doesn't throw an error anymore:
https://plnkr.co/edit/nR4Yr6iFtcoBfv1O6wZC?p=preview

Thanks!

**Comment by jrobinso** [comment]
I'm trying to merge that fix to the 1.0.x branch, if successful I'll
release 1 .0.9.   The master (beta) will be released as 1.1 at the end of
August.  It has some major updates.
